packages/agents/src/observability/index.ts

```diff
@@ -34,6 +35,13 @@ export type ObservabilityEvent =
       {
         method: string;
         streaming?: boolean;
+        success: boolean;
+      }
+    >
+  | BaseEvent<
+      "message:request" | "message:response",
+      {
+        message: UIMessage[];
```

Comment:
we probably don't want to log all messages, it'll blow past the limit on o11y events